app/views/bulkrax/entries/show.html.erb

```diff
@@ -33,7 +33,7 @@
 
     <p class='bulkrax-p-align'>
       <% if @importer.present? %>
-        <% factory_record = @entry.factory.find %>
+        <% factory_record = nil #@entry.factory.find %>
```

Comment:
what's this change about?

Reply:
it was a temporary workaround that i forgot to remove. should be good now.